### ISSUE TITLE
There is a conflict between amplify and awsiot Library

### ISSUE THREAD

**Issue by lgh89** [aws-amplify/aws-sdk-ios#3568]
There is a conflict between amplify and awsiot Library；
My podfile is as follows:
```
# Uncomment the next line to define a global platform for your project
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '13.0'
use_frameworks!

target 'Todo' do
  # Comment the next line if you don't want to use dynamic frameworks

  # Pods for Todo
  pod 'Amplify'
  pod 'AmplifyPlugins/AWSCognitoAuthPlugin'        
  pod 'AmplifyPlugins/AWSDataStorePlugin'         
  pod 'AmplifyPlugins/AWSAPIPlugin'               
  pod 'AmplifyPlugins/AWSS3StoragePlugin'         
  pod 'AmplifyPlugins/AWSPinpointAnalyticsPlugin'  
  pod 'AWSPredictionsPlugin'                      
  pod 'CoreMLPredictionsPlugin'          
  
  pod 'AWSMobileClient'
  pod 'AWSIoT'
#  pod 'AWSPinpoint'
  

  pod 'AFNetworking', '4.0.1'
  pod 'CocoaAsyncSocket', '7.4.2'
end
```

Compilation error:
In file included from /Users/lgh/Desktop/awsProject/Todo/Pods/AWSTranscribeStreaming/AWSTranscribeStreaming/Internal/AWSSRWebSocketDelegateAdaptor.m:17:
/Users/lgh/Desktop/awsProject/Todo/Pods/AWSTranscribeStreaming/AWSTranscribeStreaming/Internal/AWSSRWebSocketDelegateAdaptor.h:17:9: fatal error: 'AWSSRWebSocket.h' file not found
#import "AWSSRWebSocket.h"
        ^~~~~~~~~~~~~~~~~~
1 error generated.
In file included from /Users/lgh/Desktop/awsProject/Todo/Pods/AWSTranscribeStreaming/AWSTranscribeStreaming/Internal/AWSSRWebSocketAdaptor.m:17:
/Users/lgh/Desktop/awsProject/Todo/Pods/AWSTranscribeStreaming/AWSTranscribeStreaming/Internal/AWSSRWebSocketAdaptor.h:17:9: fatal error: 'AWSSRWebSocket.h' file not found
#import "AWSSRWebSocket.h"
        ^~~~~~~~~~~~~~~~~~
1 error generated.

Please help me to solve the problem

**Comment by lgh89** [comment]
> There is a conflict between amplify and awsiot Library；
> My podfile is as follows:
> 
> source 'https://github.com/CocoaPods/Specs.git'
> 
> platform :ios, '13.0'
> use_frameworks!
> 
> target 'Todo' do
> 
> pod 'Amplify'
> pod 'AmplifyPlugins/AWSCognitoAuthPlugin'
> pod 'AmplifyPlugins/AWSDataStorePlugin'
> pod 'AmplifyPlugins/AWSAPIPlugin'
> pod 'AmplifyPlugins/AWSS3StoragePlugin'
> pod 'AmplifyPlugins/AWSPinpointAnalyticsPlugin'
> pod 'AWSPredictionsPlugin'
> pod 'CoreMLPredictionsPlugin'
> 
> pod 'AWSMobileClient'
> pod 'AWSIoT'
> 
> pod 'AFNetworking', '4.0.1'
> pod 'CocoaAsyncSocket', '7.4.2'
> end
> 
> Compilation error:
> In file included from /Users/lgh/Desktop/awsProject/Todo/Pods/AWSTranscribeStreaming/AWSTranscribeStreaming/Internal/AWSSRWebSocketDelegateAdaptor.m:17:
> /Users/lgh/Desktop/awsProject/Todo/Pods/AWSTranscribeStreaming/AWSTranscribeStreaming/Internal/AWSSRWebSocketDelegateAdaptor.h:17:9: fatal error: 'AWSSRWebSocket.h' file not found
> #import "AWSSRWebSocket.h"
> ^~~~~~~~~~~~~~~~~~
> 1 error generated.
> In file included from /Users/lgh/Desktop/awsProject/Todo/Pods/AWSTranscribeStreaming/AWSTranscribeStreaming/Internal/AWSSRWebSocketAdaptor.m:17:
> /Users/lgh/Desktop/awsProject/Todo/Pods/AWSTranscribeStreaming/AWSTranscribeStreaming/Internal/AWSSRWebSocketAdaptor.h:17:9: fatal error: 'AWSSRWebSocket.h' file not found
> #import "AWSSRWebSocket.h"
> ^~~~~~~~~~~~~~~~~~
> 1 error generated.
> 
> Please help me to solve the problem

**Comment by palpatim** [comment]
To further clarify: I would only expect to see this happen if a customer app is including both the AWSIoT SDK *and* the AWSPredictionsPlugin

**Comment by brennanMKE** [comment]
@lgh89 I've recreated the build with your Podfile. That was very helpful. Thanks for making the repro easy. What I found is the headers which was not found is a private header in the *AWSTranscribeStreaming* framework. The import statements being used in this source file are not meant for importing headers from frameworks. I have updated them and prepared a [PR](https://github.com/aws-amplify/aws-sdk-ios/pull/3653). I will update the Podfile to point to my temporary branch to also see if there any other issues due to the same reason. I'll update this issue once I have done that check.

```sh
❯ find Build/Products/Debug-iphonesimulator -name AWSSRWebSocket.h
Build/Products/Debug-iphonesimulator/AWSTranscribeStreaming/AWSTranscribeStreaming.framework/PrivateHeaders/AWSSRWebSocket.h
Build/Products/Debug-iphonesimulator/AWSIoT/AWSIoT.framework/Headers/AWSSRWebSocket.h
```
